app/javascript/src/hyacinth_v1/components/projects/publish_targets/PublishTargetsEdit.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { startCase } from 'lodash';
+import produce from 'immer';
+
+import TabHeading from '../../shared/tabs/TabHeading';
+import { PublishTargetsForm } from './PublishTargetsForm';
+import { projects } from '../../../utils/hyacinthApi';
+import ProjectInterface from '../ProjectInterface';
+
+export default class PublishTargetsEdit extends React.Component {
+  state = {
+    project: {},
+  }
+
+  componentDidMount() {
+    const { match: { params: { projectStringKey } } } = this.props;
+
+    projects.get(projectStringKey)
```

Comment:
We're really trying not to add more references to the old REST api in the app.  Is there a reason we can't use the project graphql query?  (see: `getProjectQuery` in `app/javascript/src/hyacinth_v1/graphql/projects.js`).

Reply:
Just an oversight!

Reply:
Done!